src/gabc/gabc-write.c

```diff
@@ -491,11 +491,7 @@ static void gabc_write_gregorio_note(FILE *f, gregorio_note *note,
         }
         break;
     case S_PUNCTUM_INCLINATUM_AUCTUS:
-        if (note->next) {
-            fprintf(f, "%c<", toupper((unsigned char)pitch_letter(note->u.note.pitch)));
-        } else {
-            fprintf(f, "%c", toupper((unsigned char)pitch_letter(note->u.note.pitch)));
-        }
+        fprintf(f, "%c", toupper((unsigned char)pitch_letter(note->u.note.pitch)));
```

Comment:
Ok, merging then. This must have required a lot of patience, but it was definitely worth it, that's a very important improvement!